common/annotations/en.xml

```diff
@@ -1437,7 +1437,7 @@ annotations.
 		<annotation cp="🎅" type="tts">Santa Claus</annotation>
 		<annotation cp="🤶">celebration | Christmas | claus | mother | Mrs. | Mrs. Claus</annotation>
 		<annotation cp="🤶" type="tts">Mrs. Claus</annotation>
-		<annotation cp="🧑‍🎄">Claus, christmas | mx claus</annotation> <!-- 1F9D1 200D 1F384 -->
+		<annotation cp="🧑‍🎄">claus | christmas | Mx. Claus</annotation> <!-- 1F9D1 200D 1F384 -->
```

Comment:
Shouldn't it be mx claus without period to match the tts form? (Otherwise doesn't the tts form get automatically added by CLDRModify?)

Reply:
makes sense. fixed